Update datapunt docker

Lytrix opened the pull request:
First updated dockerized version with arguments for the 2 data dumps to separate databases (OIS and IV)

Don't accept the PR yet, Ansible also has to change to work with this PR.

Followed these guidelines:
https://amsterdam.github.io/guides/back-end-projects/

Lytrix committed:
Updated import, download, docker-compose using bash+env variables

- Removed not used tests folder, setup.py and makeFile
- Reworked downloader.py to work with objectstore variables in environment
- Changed importer.py to work with Bash script instead of makeFile
- Updated docker-compose to run download and import using environment variables
- Updated Readme

Lytrix committed:
Created arguments for specific containers queries and databases

- Removed hard coded arguments from scripts
- Better readme description with examples
- Created bash scripts for setting arguments for dev

Lytrix committed:
updated container name argument

Lytrix committed:
fix markdown spacing

Lytrix committed:
Added how to access database

jklardie commented:
Grote stap in de goede richting! Er zijn enkele punten waar ik tegenaan loop:

- `python download.py $OBJECTSTORE_CONTAINER data`. 
    - Moet volgens mij zijn `downloader.py`.

- Het download script is een beetje onduidelijk in verwachtingen:
    - Als de data/ map niet bestaat dan krijg ik een foutmelding
    - Als de data/ map wel bestaat, krijg ik een foutmelding (maar hier staat niet dat ik --overwrite moet gebruiken)
    - In je voorbeeld gebruik je data/ in de src/import/ map, maar nadat ik het script draai ziet git dit als een nieuwe folder. Om te voorkomen dat deze data (privacy gevoelig..?) in github komt misschien goed om bijvoorbeeld data/ aan gitignore toe te voegen. 

- `python importer.py data $SQL_QUERY`
    - Syntax klopt niet, moet volgens mij zijn `python importer.py -d data $SQL_QUERY`. Lijkt me goed één syntax te kiezen (args of kwargs), maar niet te mixen. 
    - Als ik de README stap voor stap volg werkt dit nog niet; DATABASE_* environment vars zijn vereist.

Lytrix committed:
ignoring relative data folder

Lytrix committed:
Improved import steps + descriptions
- Create a target directory if not exists
- Renamed python scripts to import/download
- Using kwargs for all argparse commands
- Separated database arguments as a function

Lytrix committed:
Added more envs to test in development

Lytrix committed:
Added missing key in arg

jklardie reviewed (approved):
Ziet er goed uit!
